### PR TITLE
Adds the Colt Delta Elite and accompanying cargo crates.

### PR DESCRIPTION
**Pull request by pireamaineach**
Adds the Colt Delta Elite, a 10mm version of the 1911 pistol, and adds in two related cargo crates.

Variety is the spice of life.

Delta Elite

:cl:
add: Adds the Colt Delta Elite.
add: Adds the Colt Delta Elite magazine.
add: Adds the Colt Delta Elite crate.
add: Adds the Colt Delta Elite magazine crate.
fix: Makes the 1911 magazine actually display the appropriate sprite for the amount of rounds.
imageadd: Adds the Colt Delta Elite sprites.
/:cl:

### CODE REVIEW

**Comment by pireamaineach** [comment]
My commits are fucked but eh.

File: code/modules/projectiles/guns/ballistic/pistol.dm [hunk 1]
```diff
@@ -35,6 +35,17 @@
 /obj/item/gun/ballistic/automatic/pistol/m1911/no_mag
 	spawnwithmagazine = FALSE
 
+/obj/item/gun/ballistic/automatic/pistol/deltaelite
+	name = "\improper Colt Delta Elite"
+	desc = "A modification on the classic 1911 handgun, this gun is chambered in 10mm. Like it's predecessor, it suffers from low capacity."
```
Comment (bobbahbrown) [inline code/modules/projectiles/guns/ballistic/pistol.dm]: This description should read "...Like its predecessor..."

Comment (pireamaineach) [reply code/modules/projectiles/guns/ballistic/pistol.dm]: Honestly the difference between its and it's is entirely pointless.
There's not even a reason why they shouldn't read the same.

**Review by bobbah — APPROVED** [review]
thanks

**Comment by necromanceranne** [comment]
When your cargo boys are packing burners.

**Comment by GuyonBroadway** [comment]
Suggestion, remove ranged stuns of all types and give sec these instead

**Comment by pireamaineach** [comment]
Because sec only having lethals is a grand idea.

**Comment by Tlaltecuhtli** [comment]
works in america

**Comment by pireamaineach** [comment]
They've tasers and handcuffs.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -543,6 +543,26 @@
 					/obj/item/ammo_box/magazine/wt550m9)
 	crate_name = "auto rifle ammo crate"
 
+/datum/supply_pack/security/armory/deltaelite
```
Comment (Tlaltecuhtli) [inline code/modules/cargo/packs.dm]: make it /datum/supply_pack/security/deltaelite so you can open them with normal security access

**Review by Tlaltecuhtli — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (pireamaineach) [reply code/modules/cargo/packs.dm]: Can you open Autorifle crates with security access?

Comment (pireamaineach) [reply code/modules/cargo/packs.dm]: Eh... y'know, why not?

Comment (Tlaltecuhtli) [reply code/modules/cargo/packs.dm]: you can open lasers

Comment (pireamaineach) [reply code/modules/cargo/packs.dm]: Sure that's a bit irrelevant.

**Comment by optimumtact** [comment]
we actually have zero need for another weapon, so why would we merge this?

**Comment by pireamaineach** [comment]
> we actually have zero need for another weapon, so why would we merge this?

The only reason it's an entirely different gun is to satisfy potential complaints that a classic 1911 isn't futuristic enough, and to keep the 1911 as a snowflake space gun.
It's just an option for a lower capacity, higher damage weapon from cargo, will give people more options for ballistic weapons, it's often complained that there's very few.

**Comment by optimumtact** [comment]
`it's often complained that there's very few.`

i've never once heard someone complain

**Comment by pireamaineach** [comment]
> `it's often complained that there's very few.`
> 
> i've never once heard someone complain

"orange man out of touch with playerbase"

**Comment by optimumtact** [comment]
okay, I've had it up to the top with your shitty behaviour.

There is absolutely zero reason to add your weapon and you're acting like I’m the bad guy for asking you to fill a fucking niche with your dumbass pet weapon before coming along and making your PR in the first place.

I have no obligation to sit here and listen to you lambast me because I want people to ensure things are added to actually fill a need or obligation that actually means it's worth carrying yet another sprite and weapon and fucking custom ammo type.

SO in this case, I won't be deigning to bother with your shit.

**Comment by pireamaineach** [comment]
"Bip Today at 00:02
Can I get a reason for not adding an option for ballistic weaponry if you go out of your way to get it?
It just being a restriction that exists is pretty [not cool.]"
"Confused Bi-Product of Culture Today at 00:03
I really don't give a shit if it's not cool or not"
"Confused Bi-Product of Culture Today at 00:03
I don't get to be cool"
"Bip Today at 00:04
Oranges I'm just asking for a fucking reason.
Jesus fuck."
"Confused Bi-Product of CultureToday at 00:05
I gave you the reason"
"Bip Today at 00:08
@Confused Bi-Product of Culture Hey you really didn't though, the reason you gave weren't exactly valid, as there's no reason for them to exist.
I get that someone said N.T. doesn't want the station being damaged and that's a good reason to not have them by default, but I don't see why it shouldn't be an option period."
Very good asking for justification is now making oranges the bad guy and lambasting him.

**Pull request closed without merging**